infra/dataindex_and_jobservice/03-jobservice.yml

```diff
@@ -0,0 +1,85 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: jobs-service-postgresql
+  name: jobs-service-postgresql
+spec:
+  replicas: 1
+  # remove after https://issues.redhat.com/browse/KOGITO-8311 is done
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: jobs-service-postgresql
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: jobs-service-postgresql
+    spec:
+      containers:
+        - name: jobs-service-postgresql
+          image: quay.io/kiegroup/kogito-jobs-service-allinone:latest
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+              name: http
+              protocol: TCP
+          env:
+            - name: KUBERNETES_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: QUARKUS_DATASOURCE_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  key: POSTGRES_USER
+                  name: postgres-secrets
+            - name: QUARKUS_DATASOURCE_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: POSTGRES_PASSWORD
+                  name: postgres-secrets
+          envFrom:
+            - configMapRef:
+                name: jobs-service-properties
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /q/health/live
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 0
+            periodSeconds: 30
+            successThreshold: 1
+            timeoutSeconds: 10
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /q/health/ready
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 0
+            periodSeconds: 30
+            successThreshold: 1
+            timeoutSeconds: 10
+      initContainers:
```

Comment:
Instead, can we use [Startup probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/)?

Reply:
My concern is that for prod, we can't use `busybox`. The Quarkus Health checks won't ping the postgres service?